examples/text_to_sql/gen_embed.sql

```diff
@@ -0,0 +1,16 @@
+
+\getenv ollama_host OLLAMA_HOST
+
+-- generate embeddings for the obj descriptions
+insert into ai.semantic_catalog_obj_1_store(embedding_uuid, objtype, objnames, objargs, chunk_seq, chunk, embedding)
+select
+  gen_random_uuid()
+, objtype, objnames, objargs
+, 0
+, description
+, ai.ollama_embed('nomic-embed-text', description, host=>:'ollama_host')
```

Comment:
is that colon supposed to be there? before `ollama_host`

Reply:
yes. that designates a psql variable. We loaded the OLLAMA_HOST environment variable into a psql variable named `ollama_host`